next.config.mjs

```diff
@@ -100,9 +109,12 @@ const nextConfig = {
       '@radix-ui/react-select',
       '@radix-ui/react-toast',
       'tailwindcss',
+      'shiki',
     ],
     // Removes the warning regarding the WebPack Build Worker
     webpackBuildWorker: false,
+    // Enables Next.js's Instrumentation Hook
+    instrumentationHook: true,
```

Comment:
this is neat https://nextjs.org/docs/app/api-reference/next-config-js/instrumentationHook